src/components/NcDialog/NcDialog.vue

```diff
@@ -76,6 +76,56 @@ export default {
 }
 </script>
 ```
+
+### Form example
+It is also possible to use the dialog for small forms.
+This can be used when asking for a password, a name or similar to have native form validation.
+
+To make the dialog a form the `is-form` prop needs to be set.
+When using the form variant you can also pass buttons with `nativeType` prop to add a native `submit` button.
+
+Note that this is not possible if the dialog contains a navigation!
+
+```vue
+<template>
+	<div>
+		<NcButton @click="showDialog = true">Show dialog</NcButton>
+		<NcDialog is-form
+			:buttons="buttons"
+			name="Choose a name"
+			:open.sync="showDialog"
+			@submit="currentName = newName">
+			<NcTextField label="New name"
+				placeholder="Min. 6 characters"
+				minlength="6"
+				:value.sync="newName" />
```

Comment:
Otherwise it's a bit unclear that empty name is valid

```suggestion
			<NcTextField label="New name"
				placeholder="Min. 6 characters"
				required
				minlength="6"
				:value.sync="newName" />
```